importer/tag/src/main/java/org/itsallcode/openfasttrace/importer/tag/LongTagImportingLineConsumer.java

```diff
@@ -63,13 +79,29 @@ public void processMatch(final Matcher matcher, final int lineNumber, final int
         final SpecificationItemId generatedId = SpecificationItemId.createId(matcher.group(1),
                 generatedName, 0);
 
+        final List<String> neededArtifactTypes = parseCommaSeparatedList(matcher.group(6));
+
         LOG.finest(() -> "File " + this.file + ":" + lineNumber + ": found '" + generatedId
-                + "' covering id '" + coveredId + "'");
+                + "' covering id '" + coveredId + "', needs artifact types " + neededArtifactTypes);
```

Comment:
I would log the last part only if not empty. It will be the exception that it is filled, because mostly people use it for code files.